envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -70,42 +68,88 @@ private fun getCommunicationMode(proto: Value?): CommunicationMode {
 }
 
 fun Value?.toStatusCodeFilter(accessLogFilterFactory: AccessLogFilterFactory):
-        AccessLogFilterSettings.StatusCodeFilterSettings? = this?.stringValue?.let {
+    AccessLogFilterSettings.StatusCodeFilterSettings? = this?.stringValue?.let {
```

Comment:
I know it's not your change but it took me a while to figure out that someone put the return time on a new line and used `=` to define the body.

Reply:
I added return